web/tests/EducationBenchmarking.Web.E2ETests/Pages/School/BenchmarkWorkforcePage.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Microsoft.Playwright;
+using Xunit;
+
+namespace EducationBenchmarking.Web.E2ETests.Pages.School;
+
+public enum WorkforceChartNames
+{
+    SchoolWorkforce,
+    TotalNumberOfTeacher,
+    SeniorLeadership,
+    TeachingAssistant,
+    NonClassRoomSupportStaff,
+    AuxiliaryStaff,
+    SchoolWorkforceHeadcount
+}
+
+public class BenchmarkWorkforcePage(IPage page)
+{
+    private ILocator PageH1Heading => page.Locator(Selectors.H1);
+    private ILocator BreadCrumbs => page.Locator(Selectors.GovBreadcrumbs);
+    private ILocator ChangeSchoolLink => page.Locator(Selectors.ChangeSchoolLink);
+    private ILocator ViewAsTableRadio => page.Locator(Selectors.ModeTable);
+    private ILocator ViewAsChartRadio => page.Locator(Selectors.ModeChart);
+    private ILocator Charts => page.Locator(Selectors.Canvas);
+    private ILocator SchoolWorkforceDimension => page.Locator(Selectors.SchoolWorkforceDimension);
+    private ILocator TotalNumberOfTeacherDimension => page.Locator(Selectors.TotalNumberOfTeacherDimension);
+    private ILocator SeniorLeadershipDimension => page.Locator(Selectors.SeniorLeadershipDimension);
+    private ILocator TeachingAssistantDimension => page.Locator(Selectors.TeachingAssistantDimension);
+    private ILocator NonClassRoomSupportStaffDimension => page.Locator(Selectors.NonClassRoomSupportStaffDimension);
+    private ILocator AuxiliaryStaffDimension => page.Locator(Selectors.AuxiliaryStaffDimension);
+    private ILocator SchoolWorkforceHeadcountDimension => page.Locator(Selectors.SchoolWorkforceHeadcountDimension);
+    private ILocator TotalTeachersTable => page.Locator(Selectors.Table).Nth(1);
+    private ILocator Tables => page.Locator(Selectors.Table);
+    private ILocator SaveImageSchoolWorkforce => page.Locator(Selectors.SchoolWorkforceSaveAsImage);
+
+    private ILocator SaveAsImageButtons =>
+        page.Locator(Selectors.Button, new PageLocatorOptions { HasText = "Save as image" });
+
+    public async Task IsDisplayed()
+    {
+        await PageH1Heading.ShouldBeVisible();
+        await BreadCrumbs.ShouldBeVisible();
+        await ChangeSchoolLink.ShouldBeVisible();
+        await ViewAsTableRadio.ShouldBeVisible().ShouldBeChecked(false);
+        await ViewAsChartRadio.ShouldBeVisible().ShouldBeChecked();
```

Comment:
is there a reason we are doing multiple assertions in same line and using TaskLocatorExtensions.cs method instead of LocatorExtensions method? 
Its more for me to get the understanding of doing it this way.